system/system_monitor/config/net_monitor.param.yaml

```diff
@@ -3,3 +3,5 @@
     devices: ["*"]
     traffic_reader_port: 7636
     monitor_program: "greengrass"
+    ip_reasm_fails_check_duration: 1
```

Comment:
A shortened variable name should not be used.
Please consider to use `reassemble_failed_check_duration` &  `reassemble_failed_check_count`.

Reply:
@ito-san 
I changed the word as suggested.
24a196af79dda9c44c2781252396b25fd5df32f2